charts/unspec-service/values.preview.template.yaml

```diff
@@ -43,6 +76,8 @@ ccd:
       enabled: true
     s2s:
       enabled: false
+    postgresql:
+      enabled: false
```

Comment:
set in values.yaml is it needed?

Reply:
we are creating database ourself for all needed schema instead of using one from ccd

Reply:
but the same thing is defined in values.yaml so it will propegate to here by default, my understanding is that this is just duplicate